src/internal/server/server.go.tmpl

```diff
@@ -0,0 +1,60 @@
+// This is an example file for handling web service routes.
+// When implementing the real server, make sure to create an instance of `http.Server`,
+// provide a valid configuration and apply a base context
+
+package server
+
+import (
+	"{{ .repoBaseUrl }}/{{ .serviceName }}/internal/restapi/middleware"
+	v1health "{{ .repoBaseUrl }}/{{ .serviceName }}/internal/restapi/v1/health"
+
+	"context"
+	"log"
+	"net"
+	"net/http"
+	"time"
+
+	"github.com/go-chi/chi/v5"
```

Comment:
Cool! We're now on current chi :)

Reply:
The template was using chi v5 already. I only moved the logic to a new file